diff --git a/lib/trusted-publishing/oidc-context.js b/lib/trusted-publishing/oidc-context.js
index 0427ae9de9da75861b5db715420c9e82a727fbb3..f0350f419b52bdde91f8fca6e5c91bc0a46a3bec 100644
--- a/lib/trusted-publishing/oidc-context.js
+++ b/lib/trusted-publishing/oidc-context.js
@@ -1,6 +1,10 @@
 import { OFFICIAL_REGISTRY } from "../definitions/constants.js";
 import exchangeToken from "./token-exchange.js";
 
+// registry.npmjs.com is an alias for the official registry.npmjs.org
+const OFFICIAL_REGISTRY_COM = "https://registry.npmjs.com/";
+
 export default async function oidcContextEstablished(registry, pkg, context) {
-  return OFFICIAL_REGISTRY === registry && !!(await exchangeToken(pkg, context));
+  const isOfficialRegistry = registry === OFFICIAL_REGISTRY || registry === OFFICIAL_REGISTRY_COM;
+  return isOfficialRegistry && !!(await exchangeToken(pkg, context));
 }
